packages/pn-commons/src/components/__test__/PnInfoCard.test.tsx

```diff
@@ -0,0 +1,140 @@
+import { vi } from 'vitest';
+
+import { createMatchMedia, queryByTestId } from '@pagopa-pn/pn-commons/src/test-utils';
+
+import { fireEvent, render, waitFor } from '../../test-utils';
+import PnInfoCard from '../PnInfoCard';
+
+const mockBtnOneCbk = vi.fn();
+const mockBtnTwoCbk = vi.fn();
+
+const title = 'Mocked title';
+const subTitle = 'Mocked subtitle';
+const body = <div data-testid="body">Body</div>;
+const actions = [
+  <button key="one" data-testid="btn-one" onClick={mockBtnOneCbk}>
+    One
+  </button>,
+  <button key="two" data-testid="btn-two" onClick={mockBtnTwoCbk}>
+    Two
+  </button>,
+];
+
+vi.mock('react-i18next', () => ({
+  // this mock makes sure any components using the translate hook can use it without a warning being shown
+  useTranslation: () => ({
+    t: (str: string) => str,
+  }),
+}));
+
+describe('PnInfoCard Component', () => {
+  afterEach(() => {
+    vi.clearAllMocks();
+  });
+
+  afterAll(() => {
+    vi.restoreAllMocks();
+  });
+
+  it('renders component', async () => {
+    // render component
+    const { container, getByTestId, queryByTestId } = render(
+      <PnInfoCard title={title} subtitle={subTitle} actions={actions}>
+        {body}
+      </PnInfoCard>
+    );
+    const headerEl = queryByTestId('PnInfoCardHeader');
+    expect(headerEl).toBeInTheDocument();
+    const titleEl = getByTestId('PnInfoCardTitle');
+    expect(titleEl).toBeInTheDocument();
+    expect(titleEl).toHaveTextContent(title);
+    expect(container).toHaveTextContent(subTitle);
+    const bodyEl = getByTestId('body');
+    expect(bodyEl).toBeInTheDocument();
+    expect(bodyEl).toHaveTextContent(/Body/i);
+
+    const btnOne = getByTestId('btn-one');
+    expect(btnOne).toBeInTheDocument();
+    fireEvent.click(btnOne);
+    await waitFor(() => {
+      expect(mockBtnOneCbk).toHaveBeenCalledTimes(1);
+    });
+    const btnTwo = getByTestId('btn-two');
+    expect(btnTwo).toBeInTheDocument();
+    fireEvent.click(btnTwo);
+    await waitFor(() => {
+      expect(mockBtnTwoCbk).toHaveBeenCalledTimes(1);
+    });
+  });
+
+  it('renders component - with header', () => {
```

Comment:
In what this is different from the previous one?

Reply:
I modified the second one to test the component with no actions